### DISCUSSION TITLE
有在umi框间使用的类似简单bbs留言板功能的开源代码吗？

### DISCUSSION THREAD

**Discussion by guijuzhejiang** [umijs/umi#12116] [General]
求教各位大侠，用umi做了个网站，需要加入一个类似简单bbs功能，只要发帖，上传图像，管理这些最基础的功能。有这种开源代码吗？

**Comment by fz6m** [comment]
没有，大部分使用 umi 的用户都是在企业内部、管理后台、toB、pc 端这些场景下，大部分都是对内或者租户的，toC 的 bbs 几乎没有，如果是 toB 的 bbs ，自己写一个简单的就可以，如果是 toC 的需求建议使用 nextjs 。
